styles/style.css

```diff
@@ -0,0 +1,80 @@
+h1 {
+  font-family: 'Noto Serif JP', sans-serif;
+}
+
+ul {
+  list-style-type: none;
+}
+
+nav ul li:hover {
+  text-decoration: underline;
+  color: white;
+}
+nav {
+  background-color: silver;
+}
+
+section {
+  border-bottom: 1px solid grey;
+}
+
+.navigation {
```

Comment:
like that you used fr's and gridding for the nav since I used flex, and this is a cool other way to do it